.circleci/config.yml

```diff
@@ -144,6 +144,28 @@ jobs:
           working_directory: tests/idl_gen
           command: ./run_tests.sh
 
+  test_imports:
+    docker:
+      - image: oasislabs/rust:latest
+        environment:
+          PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
is all of this needed?

Reply:
circleci requires a new top-level definition for each parallel task. the `PATH` stuff is because I couldn't figure out how to get it to source bashrc. I'm not a huge fan, but it works. Feel free to push back and ask me to change it.